admin/scripts/modules/uvod.php

```diff
@@ -171,12 +179,12 @@
 
 // načtení předmětů a form s rychloprodejem předmětů, fixme
 $o=dbQuery('
-  SELECT 
+  SELECT
     CONCAT(nazev," ",model_rok) as nazev,
     kusu_vyrobeno-count(n.id_predmetu) as zbyva,
     p.id_predmetu,
-    ROUND(p.cena_aktualni) as cena 
-  FROM shop_predmety p 
+    ROUND(p.cena_aktualni) as cena
+  FROM shop_predmety p
```

Comment:
Vypadá to, že editor ti upravuje mezery i na řádcích, na které jsi nesahal. Prosím oprav si nastavení. Commit je jinak nepřehledný a není vidět, které řádky se ho týkají a které se fakticky nemění, jen tam nastala automatická oprava. Mělo by být podchyceno i v rámci čtení před commitem změn.